packages/vaadin-template-renderer/test/vaadin-template-renderer.test.js

```diff
@@ -0,0 +1,79 @@
+import { expect } from '@esm-bundle/chai';
+import { fire, fixtureSync } from '@vaadin/testing-helpers';
+
+import '../vaadin-template-renderer.js';
+
+import { MyElement } from './my-element.js';
+import { HostElement } from './host-element.js';
+
+customElements.define('my-element', MyElement);
+customElements.define('host-element', HostElement);
```

Comment:
Please use more specific names, e.g. `template-host-element` and `template-wrapper` etc.

Also, we generally keep the custom elements definition together with base classes.